src/plugins/explore/public/components/visualizations/style_panel/axes/axes_selector.tsx

```diff
@@ -82,6 +83,17 @@ export const AxesSelectPanel: React.FC<AxesSelectPanelProps> = ({
     }
   };
 
+  useEffect(() => {
+    // Make sure to initially focus on first axis field selector if current field selection is empty
+    if (isEmpty(currentMapping)) {
+      setTimeout(() => {
```

Comment:
Nit: better to have some comment to elaborate the timeout.

Reply:
Would the comment on line 87 good enough?

Reply:
I guess the timeout is waiting for input component mount completely. And the 500ms timeout may better to have some comment.